src/InsetOrderOptimizer.cpp

```diff
@@ -254,14 +253,14 @@ void InsetOrderOptimizer::processHoleInsets()
     if (optimize_backwards)
     {
         // reverse the optimized order so we end up as near to the outline z-seam as possible
-        std::reverse(order_optimizer.polyOrder.begin(), order_optimizer.polyOrder.end());
+        //std::reverse(order_optimizer.paths.begin(), order_optimizer.paths.end());
```

Comment:
Commented out code.

Reply:
Done.

I had to comment it out because there is no std::swap for `Path`.